processor/awsapplicationsignalsprocessor/README.md

```diff
@@ -0,0 +1,130 @@
+
+### rules
+The rules section defines the rules (filters) to be applied
+
+| Name           | Description                                                                                                              | Default |
+|:---------------|:-------------------------------------------------------------------------------------------------------------------------| --- |
+| `selectors`    | List of metrics/traces dimension matchers.                                                                               |  [] |
+| `action`       | Action being applied for the specified selector. `keep`, `drop`, `replace`                                               |  "" |
+| `rule_name`    | (Optional) Name of rule.                                                                                                 |  [] |
+| `replacements` | (Optional) List of metrics/traces replacements to be executed. Based on specified selectors. requires `action = replace` |  [] |
+
+#### selectors
+A selectors section defines a matching against the dimensions of incoming metrics/traces.
+
+| Name        | Description                                                   | Default |
+|:------------|:--------------------------------------------------------------| ------ |
+| `dimension` | Dimension of metrics/traces                                   |   ""    |
+| `match`     | glob used for matching values of dimensions                   |   ""   |
+
+### replacements
+A replacements section defines a matching against the dimensions of incoming metrics/traces for which value replacements will be done. action must be `replace`
+
+| Name               | Description                                   | Default |
+|:-------------------|:----------------------------------------------| ------ |
+| `target_dimension` | Dimension to replace                          |   ""   |
+| `value`            | Value to replace current dimension value with |   ""   |
+
+
+## AWS AppSignals Processor Configuration Example
+
+```yaml
+awsapplicationsignals:
+    resolvers: ["eks"]
+    rules:
+      - selectors:
+          - dimension: Operation
+            match: "POST *"
+          - dimension: RemoteService
+            match: "*"
+        action: keep
+        rule_name: "keep01"
+      - selectors:
+           - dimension: Operation
+             match: "GET *"
+           - dimension: RemoteService
+             match: "*"
+        action: keep
+        rule_name: "keep02"
+      - selectors:
+           - dimension: Operation
+             match: "POST *"
+        action: drop
+        rule_name: "drop01"
+      - selectors:
+           - dimension: Operation
+             match: "*"
+        replacements:
+          - target_dimension: RemoteOperation
+            value: "This is a test string"
+        action: replace
+        rule_name: "replace01"
+```
+
+## Amazon CloudWatch Agent Configuration Example
```

Comment:
Do we still want this in here? If the idea is to separate the processor from the agent, then I don't think the agent configuration should be part of the README.